verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -234,7 +234,7 @@ std::vector<verible::lsp::Location> SymbolTableHandler::FindDefinitionLocation(
     LOG(ERROR) << "Could not find buffer with URI " << params.textDocument.uri;
     return {};
   }
-  const verilog::ParsedBuffer *parsedbuffer = tracker->current();
+  const auto parsedbuffer = tracker->current();
```

Comment:
Given that  finding any symbol is better than finding none at all, using `last_good()` might probably be a better choice. Leaving it to @glatosinski to consider.

Reply:
`parsedbuffer` in general should always exist when a file was properly opened in the LS context.
We use it to obtain the current state of the buffer,  its `TextStructureView` to be more specific.

The reason for using `current()` instead for `last_good()` is that the user can make some extensive changes in the file which are not necessarily good from the parser perspective, but can significantly change the text structure.

Due to the fact that `textDocument/definition` request sends only URI to file and symbol coordinates, the usage of `last_good` could result in extracting the wrong symbol (because the contents changed).